test/metrics/prometheus.js

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2018 Jaeger Author.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+import { assert } from 'chai';
+import PrometheusMetricsFactory from '../../src/metrics/prometheus';
+import PromClient from 'prom-client';
```

Comment:
can this be made conditional on the presence of env variable? Or, alternatively, just be conditional and if the module is not available then mark all tests as skipped.

Reply:
I suggest going with the env var scenario. And, since `0.10` is the exception not the standard, seems to make sense for the test to run by default and for the env var to indicate the test should be skipped.

Reply:
Why we have to do `import` conditionally? 
I mean `test/metrics/prometheus.js` will be skipped depending on the env var in script like travis.yml.
Do you mean this test always runs regardless of prom-client installation?